plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveWriterFactory.java

```diff
@@ -582,12 +585,9 @@ public HiveWriter createWriter(Page partitionColumns, int position, OptionalInt
                 tempFilePath = new Path(path.getParent(), ".tmp-sort." + path.getName());
             }
             try {
-                Configuration configuration = new Configuration(outputConf);
-                // Explicitly set the default FS to local file system to avoid getting HDFS when sortedWritingTempStagingPath specifies no scheme
```

Comment:
There is not equivalent of this code. It was added as part of https://github.com/trinodb/trino/commit/10fa05b8a6cd049b8b414261ae79b8e6812d49bf.
@raunaqmorarka can it be removed?

Maybe we can just prefix `sortedWritingTempStagingPath` with `file:///` when it doesn't specify schema? Right now default is `/tmp/presto-${USER}`.